packages/react/src/converters/ReactStackTraceConverter.ts

```diff
@@ -0,0 +1,51 @@
+import { BacktraceStackTraceConverter } from '@backtrace/sdk-core';
+import { JavaScriptEngine } from '@backtrace/sdk-core/src/model/data/JavaScriptEngine';
+import { BacktraceStackFrame } from '@backtrace/sdk-core/lib/model/data/BacktraceStackTrace';
+
+export class ReactStackTraceConverter implements BacktraceStackTraceConverter {
+    constructor(private readonly stackTraceConverter: BacktraceStackTraceConverter) {}
+
+    get engine(): JavaScriptEngine {
+        return this.stackTraceConverter.engine;
+    }
+
+    public convert(stackTrace: string, message = ''): BacktraceStackFrame[] {
+        // React 16 component stacks are not JS error stacks, and need to be parsed separately
+        if (this.isReact16ComponentStack(stackTrace)) {
+            return this.parseReact16ComponentStack(stackTrace);
+        }
+        return this.stackTraceConverter.convert(stackTrace, message);
+    }
+
+    /** React 16 component stacks need to be parsed separately. React 17+ component stacks can be parsed like Error stacks */
```

Comment:
the comment format is different. Can you use
```
/**
*
*/
```

Reply:
Fixed